scalanative-stubs/src/main/scala/scala/scalanative/testinterface/PreLoadedClassLoader.scala

```diff
@@ -0,0 +1,9 @@
+package scala.scalanative.testinterface
+
+/** A `ClassLoader` that returns pre-instanciated objects.
+ *  We use it to bypass uses of reflection in the test interface.
+ */
+class PreLoadedClassLoader(map: Map[String, AnyRef]) extends ClassLoader {
```

Comment:
Given that `preload` is a single word, the name should be `PreloadedClassLoader`.